### ISSUE TITLE
[ENHANCEMENT] Update the formatting of the help manual

### ISSUE THREAD

**Issue by junnunkarim** [kyoheiu/felix#284]
Can you update the help manual to show different sections like your website and make it so that it behaves like navigating with `less`?

**Comment by kyoheiu** [comment]
Yes, that is one way to show the manual, which I think a lot of CLI apps implement.
But is it really necessary to read the manual in the `less` style, especially when there is not so much texts like in felix?
If I'm going to implement this I may need to introduce another crate, which is not always a good option, so we need to have others' opinion on this.

**Comment by junnunkarim** [comment]
@kyoheiu 
On second thought, I now also think that adding a new crate might not be a good idea to view just some small amount of text.
But how about just formatting the output of the manual? Like adding sections so that it's easy to find keymaps?
